src/relay/WalletSDKRelay.ts

```diff
@@ -1046,14 +1046,11 @@ export class WalletSDKRelay extends WalletSDKRelayAbstract {
         })
 
         const _cancel = (error?: Error) => {
-          if (error) {
+          if (error && (error instanceof ErrorResponse)) {
```

Comment:
```suggestion
          if (error instanceof ErrorResponse) {
```
a falsy value won't be an instanceof anything